docs/source/versions.rst

```diff
@@ -3,6 +3,36 @@ Versions
 
 Depending on the Mesh Server version you intend to communicate with a compatible version of Mesh Python SDK is needed.
 
+`Mesh Python SDK version 1.1.0 (Release Candidate) <https://github.com/Volue-Public/energy-mesh-python/releases/tag/v1.1.0>`_
+*****************************************************************************************************************************
+
+------------
+
+Compatible with
+~~~~~~~~~~~~~~~~~~
+
+- Mesh server version >= 2.6.1
+- Python [3.7.1, 3.8 3.9 and 3.10]
+- Tested with Mesh server version 2.6.1.8
+
+New features
+~~~~~~~~~~~~~~~~~~
+
+- Support for Python 3.10 :pull:`93`
+
+.. warning::
+    Python 3.7.1 support will dropped in the next Mesh Python SDK release.
```

Comment:
With new Python 3.11 and Python 3.7.1 end of life in couple of months (June 2023) I think we can already warn the users about dropping this in the next release. Support for 3.7 complicates things with out dependencies, so I prefer to go from 3.8 and try supporting 3.11 when possible.

Reply:
Agreed - we can probably have a "session" every 6 month including new Python versions and dropping older Python version that is close to end-of-life (within the next 6 months). This strategy should be documented in the version description.